keyform/models.py

```diff
@@ -89,3 +123,25 @@ class Meta:
 
     def __str__(self):
         return str(self.created_timestamp)
+
+
+class Contact(models.Model):
+    building = models.ManyToManyField(Building, blank=False)
```

Comment:
Generally, many-to-many fields are plural.

Also you should redo all the migrations.